VidCoder/Services/ProcessingService.cs

```diff
@@ -3335,7 +3544,7 @@ private static IList<SourceSubtitleTrack> ChooseSubtitlesFromLanguages(IList<Sou
 		/// <param name="job">The job to pick the range on.</param>
 		/// <param name="title">The title the job is applied to.</param>
 		/// <param name="picker">The picker to use to pick the range.</param>
-		private void AutoPickRange(VCJob job, SourceTitle title, Picker picker = null)
+		private void AutoPickRange(string cli_timespan, VCJob job, SourceTitle title, Picker picker = null)

```

Comment:
I don't think you need to edit this method. You'd be better off with a different method like SetRangeFromString and passing the job/title/string to that. Then only call that method when the user has supplied the override on the CLI